plaso/lib/aul/statedump.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+"""The Apple Unified Logging (AUL) Statedump chunk parser."""
+import os
+import plistlib
+
+from dfdatetime import apfs_time as dfdatetime_apfs_time
+
+from plaso.helpers.macos import location
+
+from plaso.lib.aul import time as aul_time
+
+from plaso.lib import dtfabric_helper
+from plaso.lib import errors
+
+from plaso.parsers import aul
+from plaso.parsers import logger
+
+class StatedumpParser(dtfabric_helper.DtFabricHelper):
```

Comment:
add lines between imports and class definition and remove lines between plaso imports